pkg/registry/pod/etcd/etcd.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/kubernetes/pkg/registry/generic"
 	etcdgeneric "k8s.io/kubernetes/pkg/registry/generic/etcd"
 	genericrest "k8s.io/kubernetes/pkg/registry/generic/rest"
+	nodeetcd "k8s.io/kubernetes/pkg/registry/node/etcd"
```

Comment:
I know that modifying "finished" interface is a violation of an open-closed principle, but wouldn't it be problematic to add new methods to this interface, if it will be defined in multiple places? I obviously have no idea how golang imports and dependencies work...